rakelib/utils.rake

```diff
@@ -29,4 +46,33 @@ class String
     return false if self.empty? || self =~ (/^(false|f|no|n|0)$/i)
     raise ArgumentError.new "invalid value: #{self}"
   end
+
+  # colorization
+  def black;          colors_available? ? "\e[30m#{self}\e[0m" : self end
```

Comment:
We could also just have a giant `STYLES` hash with `:red => 31`, `:bg_red => 41` and `:bold => 1` all in the same hash and just `def colorize(*styles)`, would even be simpler.